tests/e2e/specs/stories-editor/page-attachment.js

```diff
@@ -0,0 +1,167 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	createNewPost,
+	saveDraft,
+} from '@wordpress/e2e-test-utils';
+
+/**
+ * Internal dependencies
+ */
+import {
+	activateExperience,
+	clickButton,
+	deactivateExperience,
+	insertBlock,
+	searchForBlock, openPreviewPage,
+} from '../../utils';
+
+/**
+ * Opens the first found Page Attachment.
+ *
+ * @return {Promise<void>} Promise.
+ */
+async function openPageAttachment() {
```

Comment:
Shouldn't these functions be utils?

Reply:
Should they be used outside this test suite at some point we can make them top level utils.